warcry_proj/README.txt

```diff
@@ -0,0 +1,22 @@
+warcry_proj README
+==================
+
+Getting Started
+---------------
+
+- Login to Vagrant
+* vagrant up
+* vagrant ssh
+
+- Ensure the bootstrap script is run to configure and install all required packages
+* sudo script/bootstrap vagrant
```

Comment:
Yeah, provisioning only runs when the vagrant box is first created. It won't run every time you vagrant up. If you want to rerun provisioning, you run "vagrant provision"